app/views/layouts/_navbar.html.haml

```diff
@@ -25,9 +34,10 @@
           View sites
 
       .btn-group
-        %button.btn.btn-primary.navbar-btn{ data: { behavior: 'export', toggle: 'tooltip', placement: 'top', container: 'body' }, title: 'Click here to export data for visible sites' }
-          %i.fa.fa-cloud-download
-          Export data
+        - if signed_in?
```

Comment:
We also don't want to hide this button if a user is not logged in, eventually we want the system to take them through the process of logging as part of the export